src/traces/parcoords/parcoords.js

```diff
@@ -456,16 +456,16 @@ module.exports = function(root, svg, parcoordsLineLayers, styledData, layout, ca
 
     parcoordsLineLayer
         .each(function(d) {
-
             if(d.viewModel) {
-                if(d.lineLayer) d.lineLayer.update(d);
-                else d.lineLayer = lineLayerMaker(this, d);
+                if((!d.lineLayer) ||
```

Comment:
That extra `( )` style feels somewhat to our codebase. Maybe we could add https://eslint.org/docs/rules/no-extra-parens to lintinting rules.